### PR TITLE
feat(Grunt Protractor): allow running two parallel test:ci processes

### PR DESCRIPTION
**Pull request by carmelc**
Allow running two parallel test:ci processes
Also, allow running test:ci on osx provided that you have the env variables defined:
SAUCE_BROWSERS
SAUCE_USERNAME
SAUCE_ACCESS_KEY

usage: from project root directory use:
OSX - node_modules/wix-gruntfile/scripts/parallel_ci_osx.sh
Unix - node_modules/wix-gruntfile/scripts/parallel_ci.sh

### CODE REVIEW

File: protractor-teamcity-conf-part2-tunnel2.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* global process, exports, jasmine */
+'use strict';
+
+var config = require('./protractor-teamcity-conf').config;
+var arrays = require('./protractor-teamcity-conf').arrays;
+
+for (var i = 0 ; i < arrays.cap2.length ; i++) {
```
Comment (shahata) [inline protractor-teamcity-conf-part2-tunnel2.js]: what's this loop for?

Comment (shahata) [inline protractor-teamcity-conf-part2-tunnel2.js]: never mind, I get it

**Comment by shahata** [comment]
Nice work!

**Comment by carmelc** [comment]
Thanks, my first ever batch script/ grunt script so was expecting way more
comments :)

**Pull request closed without merging**